larky/src/test/resources/vendor_tests/pycryptodome/Crypto/Hash/test_hash.star

```diff
@@ -0,0 +1,62 @@
+load("@stdlib//builtins", builtins="builtins")
+load("@stdlib//unittest", "unittest")
+load("@vendor//asserts", "asserts")
+load("@vendor//Crypto/Hash/MD5", MD5="MD5")
+load("@vendor//Crypto/Hash/SHA1", SHA1="SHA1")
+load("@vendor//Crypto/Hash/SHA256", SHA256="SHA256")
+load("@vendor//Crypto/Hash/SHA512", SHA512="SHA512")
+load("@vendor//Crypto/Hash/SHA3_256", SHA3_256="SHA3_256")
+load("@vendor//Crypto/Util/py3compat", tobytes="tobytes", bord="bord", tostr="tostr")
+
+
+def b(s):
+    return builtins.bytes(s, encoding="ISO-8859-1")
```

Comment:
Interesting. This encoding is important to discuss. pycryptodome does want to make sure that iso8859-1 / ascii encoding vs utf8. Let's make sure we're understanding that appropriately.

Reply:
I am using the encoding format py3compat uses. Both work since we only deal with regular unicode in ops. Is there anymore problem here? Can we merge this pr? I have sync it with your master changes